readme.md

```diff
@@ -0,0 +1,39 @@
+# City Temperature Management API
+This project is built on FastAPI and provides functionalities related to managing cities and temperature records.
+
+To clone this project from GitHub, follow these steps:
+1. Open your terminal or command prompt.
+2. Navigate to the directory where you want to clone the project.
+3. Run the following command:
+```shell
+git clone https://github.com/ArturPoltser/py-fastapi-city-temperature-management-api.git
+cd py-fastapi-city-temperature-management-api
+python -m venv venv
+venv\Scripts\activate
+```
+4. Install requirements:
+```shell
+pip install -r requirements.txt
+```
+5. You should register on weatherapi.com and get your WEATHER_API_KEY
+6. Copy .env.sample and create .env file. After that you need to put your own WEATHER_API_KEY instead of default one.
+## Adding Migrations
+This project uses Alembic for managing database migrations. Execute the following commands to create and run migration:
+```shell
+alembic revision --autogenerate -m "initial_migrations"
+alembic upgrade head
+```
+## Running the Server
+To run the server execute the following command:

```

Comment:
Which command ?

Reply:
Still not resolved!

Reply:
Added this change on Jun 9, 2024, please check the commit called "ManagementUpdate3"

Reply:
> Still not resolved!
Added this change on Jun 9, 2024, please check the commit called "ManagementUpdate3"